src/modules/private-share-folder/private-sharing.repository.ts

```diff
@@ -89,7 +89,7 @@ export class SequelizePrivateSharingRepository
     );
   }
 
-  async findPrivateFolderRoleByUserIdAndFolderId(
+  async findPrivateFolderRoleByFolderUuidAndUserUuid(
```

Comment:
Hey @cg27shokworks, I will use userId and `folderId` as you are finding by, in this table, where the fields are called like that. It is precisely due to this and for maintenance reasons that using that naming would be better than this. I find the typing on TypeScript descriptive enough to reveal that currently, the value of those fields are the `user.uuid` and the `folder.uuid`